specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2023-03-01/scheduledActions.json

```diff
@@ -0,0 +1,931 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/WeeksOfMonth"
+          }
+        },
+        "dayOfMonth": {
+          "type": "integer",
+          "description": "UTC day on which cost analysis data will be emailed. Must be between 1 and 31. This property is applicable when frequency is Monthly and overrides weeksOfMonth or daysOfWeek.",
+          "format": "int32"
+        },
+        "startDate": {
+          "description": "The start date and time of the scheduled action (UTC).",
+          "type": "string",
+          "format": "date-time"
+        },
+        "endDate": {
+          "description": "The end date and time of the scheduled action (UTC).",
+          "type": "string",
+          "format": "date-time"
+        }
+      }
+    },
+    "WeeksOfMonth": {
+      "type": "string",
+      "description": "Weeks of month.",
+      "enum": [
+        "First",
+        "Second",
+        "Third",
+        "Fourth",
+        "Last"
+      ],
+      "x-ms-enum": {
+        "name": "WeeksOfMonth",
+        "modelAsString": true
+      }
+    }
+  },
+  "parameters": {
+    "scopeParameter": {
+      "name": "scope",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The scope associated with scheduled action operations. This includes 'subscriptions/{subscriptionId}' for subscription scope, 'subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}' for resourceGroup scope, 'providers/Microsoft.Billing/billingAccounts/{billingAccountId}' for Billing Account scope, 'providers/Microsoft.Billing/billingAccounts/{billingAccountId}/departments/{departmentId}' for Department scope, 'providers/Microsoft.Billing/billingAccounts/{billingAccountId}/enrollmentAccounts/{enrollmentAccountId}' for EnrollmentAccount scope, 'providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}' for BillingProfile scope, 'providers/Microsoft.Billing/billingAccounts/{billingAccountId}/invoiceSections/{invoiceSectionId}' for InvoiceSection scope, 'providers/Microsoft.CostManagement/externalBillingAccounts/{externalBillingAccountName}' for External Billing Account scope and 'providers/Microsoft.CostManagement/externalSubscriptions/{externalSubscriptionName}' for External Subscription scope. Note: Insight Alerts are only available on subscription scope.",
+      "x-ms-parameter-location": "method"
+    },
+    "scheduledActionNameParameter": {
+      "name": "name",
+      "in": "path",
+      "description": "Scheduled action name.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "IfMatchOptionalParameter": {
+      "name": "If-Match",
+      "in": "header",
+      "required": false,
+      "description": "ETag of the Entity. Not required when creating an entity, but required when updating an entity.",
+      "type": "string"
+    }
```

Comment:
I don't know why it is different from the latest version of `2022-10-01` (see [here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/scheduledActions.json#L929)). Please add back `"x-ms-parameter-location": "method"`.

Reply:
@MattEberhart-MSFTE could you also check other files content?

Reply:
As discussed offline, added all changes that were missing by going through the equivalent files in 2022-10-01 in main branch to add anything that was added after I branched off.